suite-common/wallet-core/src/device/deviceActions.ts

```diff
@@ -2,15 +2,26 @@ import { createAction } from '@reduxjs/toolkit';
 
 import { Device, DEVICE } from '@trezor/connect';
 import { ButtonRequest, TrezorDevice } from '@suite-common/suite-types';
+import { WalletType } from '@suite-common/wallet-types';
 
 export const DEVICE_MODULE_PREFIX = '@suite/device';
 
-const connectDevice = createAction(DEVICE.CONNECT, (payload: Device) => ({ payload }));
+export type ConnectDeviceSettings = {
+    defaultWalletLoading: WalletType;
+    isViewOnlyModeVisible: boolean;
+};
```

Comment:
I'd move this to @suite-common/wallet-types

Reply:
This is basically a payload type of the action, I think it shall be here next to the action it types